content/custom_resources/about.md

```diff
@@ -242,32 +236,9 @@ matters!
 Use the **package** resource to install httpd:
 
 ```ruby
-package 'httpd' do
-  action :install
-end
+package 'httpd' # Ommiting the action uses the default action and properties on the resource
```

Comment:
I feel like it would be better to write out the full implementation then show the short after:

```
# These are equivalent 
package 'httpd' do
  action :install
end

package 'httpd' # Ommiting the action uses the default action and properties on the resource
```

Reply:
👍🏼 Agreed